src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -950,6 +951,26 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	margin: 0;
 }
 
+.interactive-list > .monaco-list > .monaco-scrollable-element > .monaco-list-rows > .monaco-list-row > .monaco-tl-row > .monaco-tl-contents {
+	/* The default is `display: block`. Allow the disabled-overlay element to show up on top of the rowContainer it applies to */
+	display: flex;
```

Comment:
This element is managed by the tree and we should avoid styling it, why does this help with what the comment says?

Reply:
We style other tree internals in this css file. Without this, the overlay element (which is a sibling of the row container content) renders below the content